### PR TITLE
Fix: Info-Box CTA padding issue on front-end

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
CTA > Button - Padding is 10 px on front end but in editor no value is displayed - 
https://share.bsf.io/kpu48qWA 

### Types of changes
non-breaking change 

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -44,7 +44,7 @@
 *.pyc binary
 
 # Ignore Git Commands
-package-lock.json 	-diff
+# package-lock.json 	-diff
```
Comment (sandeshjangam) [inline .gitattributes:47]:
```suggestion
package-lock.json 	-diff
```

**Review by sandeshjangam — APPROVED** [review]

File: includes/blocks/info-box/frontend.css.php [hunk 2]
```diff
@@ -53,7 +53,6 @@
 	),
 	' .uagb-ifb-content > svg'                             => array(
 		'height'        => $icon_size,
-		'width'         => $icon_size,
```
Comment (sandeshjangam) [inline includes/blocks/info-box/frontend.css.php:56]: WIll it create a problem for majority of icons

Comment (AbhijitNage123) [reply includes/blocks/info-box/frontend.css.php:56]: No, it is not. I tested multiple info-box on-page and with different icons and different sizes, it's working fine.

File: .github/workflows/code-analysis.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ jobs:
```
Comment (RajkiranBagal) [inline .github/workflows/code-analysis.yml:15]: WHy this change?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: includes/blocks/info-box/frontend.css.php (hunk 2 above)
Comment (RajkiranBagal) [reply includes/blocks/info-box/frontend.css.php:56]: @AbhijitNage123 Why this change is done? Let's discuss

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: .github/workflows/code-analysis.yml (hunk 3 above)
Comment (AbhijitNage123) [reply .github/workflows/code-analysis.yml:15]: We are facing an issue while code analysis action. @sandeshjangam did that change to resolve the action issue.

Comment (RajkiranBagal) [reply .github/workflows/code-analysis.yml:15]: @AbhijitNage123 Remove this change from this PR & do it in a separate PR with proper description.

File: .github/workflows/code-analysis.yml [hunk 4]
```diff
@@ -33,15 +33,18 @@ jobs:
             path: ~/.npm
             key: ${{ runner.os }}-node-${{ matrix.node }}-npm-cache-${{ hashFiles('**/package-lock.json') }}
 
+      - name: Install npm dependencies
+        run: npm install
```
Comment (RajkiranBagal) [inline .github/workflows/code-analysis.yml:37]: @sandeshjangam Should this be npm ci ?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by sandeshjangam — APPROVED** [review]

File: .github/workflows/code-analysis.yml (hunk 4 above)
Comment (sandeshjangam) [reply .github/workflows/code-analysis.yml:37]: Yes will update it, it is causing an issue. Will fix it.

File: .github/workflows/code-analysis.yml (hunk 3 above)
Comment (sandeshjangam) [reply .github/workflows/code-analysis.yml:15]: I Have added it. It wasn't replicated in different PR.

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**